ci-operator/step-registry/openshift-qe/installer/aws/sdn/fips/OWNERS

```diff
@@ -0,0 +1 @@
+../../OWNERS
```

Comment:
something is wrong with the owners file here, be sure theres an owners file at each folder you added